File: An2/MPP/mpp-proiect-java-DeepSmeag/modelModule/src/main/java/contests/model/InscrieriProba.java
package contests.model;

import contests.model.entity.Entitate;

import java.io.Serializable;

public class InscrieriProba extends Entitate<Integer> implements Serializable {
    private int nrParticipanti;
    private String numeProba;
    private String categorieVarsta;

    public InscrieriProba(int id, int nrParticipanti, String numeProba, String categorieVarsta) {
        super(id);
        this.nrParticipanti = nrParticipanti;
        this.numeProba = numeProba;
        this.categorieVarsta = categorieVarsta;
    }

    public int getNrParticipanti() {
        return nrParticipanti;
    }

    public String getNumeProba() {
        return numeProba;
    }

    public String getCategorieVarsta() {
        return categorieVarsta;
    }

    public String toString() {
        return "InscrieriProba{" +
                "nrParticipanti=" + nrParticipanti +
                ", numeProba='" + numeProba + '\'' +
                ", categorieVarsta='" + categorieVarsta + '\'' +
                '}';
    }
}

File: An2/MPP/mpp-proiect-csharp-DeepSmeag/domain/InscrieriProba.cs
using mpp_proiect_csharp_DeepSmeag.domain.entity;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace mpp_proiect_csharp_DeepSmeag.domain
{
    internal class InscrieriProba: Entitate<int>
    {
        public int nrParticipanti { get; }
        public string numeProba { get; }
        public string categorieVarsta { get; }

        public InscrieriProba(int id, int nrParticipanti, string numeProba, string categorieVarsta) : base(id)
        {
            this.nrParticipanti = nrParticipanti;
            this.numeProba = numeProba;
            this.categorieVarsta = categorieVarsta;
        }

        public String toString()
        {
            return "InscrieriProba{" +
                    "nrParticipanti=" + nrParticipanti +
                    ", numeProba='" + numeProba + '\'' +
                    ", categorieVarsta='" + categorieVarsta + '\'' +
                    '}';
        }

    }
}
